mmv1/products/bigquery/terraform.yaml

```
@@ -41,6 +41,14 @@ overrides: !ruby/object:Overrides::ResourceOverrides
           private: "private"
           public: "public"
           account_name: "bqowner"
+      - !ruby/object:Provider::Terraform::Examples
+        name: "bigquery_dataset_authorized_routine"
+        primary_resource_id: "dataset"
```

This needs to match the second part of a `dataset` resource's terraform id. For example, for:

```hcl
resource "google_bigquery_dataset" "public" {
    // args
}
```

the resource address is `google_bigquery_dataset.public`, so primary_resource_id would need to be `public`.

The "primary resource" will be imported as part of the test. In this case, the important thing is to make sure that imports work correctly for the `routine` field (which is on the `private` dataset)

```suggestion
        primary_resource_id: "private"
```